ui/appui-react/src/appui-react/accudraw/AccuDrawFieldContainer.tsx

```diff
@@ -206,6 +213,28 @@ export function AccuDrawFieldContainer(props: AccuDrawFieldContainerProps) {
     []
   );
 
+  const focusNextField = React.useCallback((field: ItemField) => {
```

Comment:
I think we can get rid of `ItemField` param in `onTabPressed` to align with other handlers. This switch can be removed (code blocks can simply be inlined in their respective `AccuDrawInputField` elements.